app/screens/channel/channel_base.js

```diff
@@ -37,6 +37,10 @@ export default class ChannelBase extends PureComponent {
         showTermsOfService: PropTypes.bool,
         skipMetrics: PropTypes.bool,
         viewingGlobalThreads: PropTypes.bool,
+        selectedPost: PropTypes.shape({
+            id: PropTypes.string,
+            channel_id: PropTypes.string,
```

Comment:
this two should be required if `selectedPost` is set, right ?

Reply:
oh, yes, will add that now :)